src/pages/LEVANTE/UserSurvey.vue

```diff
@@ -10,6 +10,42 @@ import { useRouter } from 'vue-router';
 import { useGameStore } from '@/store/game';
 import { Converter } from 'showdown';
 import { useI18n } from 'vue-i18n';
+import { BufferLoader, AudioContext } from '@/helpers/audio';
+
+const generateAudioLinks = (parsedLocale) => {
+  const fileNames = [
+    'ChildSurveyIntro',
+    'ClassFriends',
+    'ClassHelp',
+    'ClassNice',
+    'ClassPlay',
+    'Example1Comic',
+    'Example2Neat',
+    'GrowthMindMath',
+    'GrowthMindRead',
+    'GrowthMindSmart',
+    'LearningGood',
+    'LonelySchool',
+    'MathEnjoy',
+    'MathGood',
+    'ReadingEnjoy',
+    'ReadingGood',
+    'SchoolEnjoy',
+    'SchoolFun',
+    'SchoolGiveUp',
+    'SchoolHappy',
+    'SchoolSafe',
+    'TeacherLike',
+    'TeacherListen',
+    'TeacherNice',
+  ];
+
+  const baseURL = `https://storage.googleapis.com/road-dashboard/child-survey/${parsedLocale}/shared/`;
```

Comment:
Instead of hardcoding these, could we get them directly from the bucket? You can do a request to list out all the objects in a bucket and them parse them by the folder they live in. 

```suggestion
  `https://storage.googleapis.com/storage/v1/b/${bucketName}/o`
```

Reply:
Adding on to this, this way we could also use this for teacher and parent questionnaires.

Reply:
I think I have set it up in the way where this is configurable. But not sure how to pass that in. Maybe we should do it as a separate PR?

Reply:
Agreed. This is good for now.